src/main/java/seedu/address/logic/parser/SortCommandParser.java

```diff
@@ -0,0 +1,35 @@
+package seedu.address.logic.parser;
+
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import java.util.Arrays;
+import java.util.List;
+
+import seedu.address.logic.commands.SortCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new SortCommand object
+ */
+public class SortCommandParser implements Parser<SortCommand> {
+    private String[] categories = {"name", "phone", "email", "address"};
+    private List<String> categoriesList = Arrays.asList(categories);
```

Comment:
I like @cmang12 's suggestion on encapsulating the categories, or perhaps consider making the list immutable, or consist of immutable constants? Should it also be a consideration to avoid magic strings?